src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs

```diff
@@ -0,0 +1,556 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+// Author:
```

Comment:
mm all I changed in the tests themselves was test framework related, so basically porting them to xunit, but functioning-wise I didn't change a thing.